src/gulpfile.js

```diff
@@ -255,6 +255,29 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor', 'add-revision-numbers'], function () {
+  var sources = gulp.src([
+    paths.build + '/template-editor/relayEditor.*.js'
+  ]);
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(sources, {
+    addRootSlash: false, // ensures proper relative paths
+    ignorePath: paths.build + '/template-editor' // ensures proper relative paths
+  }))
+  .pipe(gulp.dest(paths.build + '/template-editor'));
+});
+
+gulp.task('template-editor', function () {
+  return gulp.src([
+    paths.app + '/template-editor/adapterService.js',
+    paths.app + '/env/' + process.env.NODE_ENV + '.js'
```

Comment:
Does it work? or throw a kind of error because the module does not exist as you said to me before?

Reply:
It is throwing this error:
![image](https://user-images.githubusercontent.com/7245667/41300637-9a6fa43a-6e3c-11e8-985b-02319278cd3c.png)

Reply:
In that case, I prefer removing this line or fix the error before the merge.

Reply:
Perfect. I will merge it, after to take cover the dependency task in DR-1649